client/allocrunner/csi_hook.go

```diff
@@ -231,3 +236,25 @@ func (c *csiHook) shouldRun() bool {
 
 	return false
 }
+
+func (c *csiHook) unmount(pair *volumeAndRequest) error {
+
+	mounter, err := c.csimanager.MounterForPlugin(context.TODO(), pair.volume.PluginID)
+	if err != nil {
+		return err
+	}
+
+	usageOpts := &csimanager.UsageOptions{
+		ReadOnly:       pair.request.ReadOnly,
+		AttachmentMode: pair.request.AttachmentMode,
+		AccessMode:     pair.request.AccessMode,
+		MountOptions:   pair.request.MountOptions,
+	}
+
+	err = mounter.UnmountVolume(context.TODO(),
+		pair.volume.ID, pair.volume.RemoteID(), c.alloc.ID, usageOpts)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
personal style, feel free to ignore
```suggestion
	return mounter.UnmountVolume(context.TODO(),
		pair.volume.ID, pair.volume.RemoteID(), c.alloc.ID, usageOpts)
```